packages/app-project/src/components/ProjectHeader/ProjectHeader.stories.js

```diff
@@ -0,0 +1,267 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Grommet } from 'grommet'
+import { Provider } from 'mobx-react'
+import { applySnapshot } from 'mobx-state-tree'
+import { RouterContext } from 'next/dist/shared/lib/router-context'
+
+import initStore from '@stores'
+import ProjectHeader from './ProjectHeader.js'
+import * as subcomponents from './components'
+
+const store = initStore(false)
```

Comment:
Probably safer to scope the store to each story, rather than use a single store for all the stories. That will probably fix the Docs page error.